### PR TITLE
Fix broken link to notebook in parameter module

### PR DESCRIPTION
**Pull request by lakhotiaharshit**
Fixes a broken link to the notebook in the parameter module

@astafan8

### CODE REVIEW

File: qcodes/instrument/parameter.py [hunk 1]
```diff
@@ -27,7 +27,7 @@
 - :class:`.ParameterWithSetpoints` is intended for array-values parameters.
     This Parameter class is intended for anything where a call to the instrument
     returns an array of values.
-    `This notebook <../examples/writing_drivers/Simple-Example-of-ParameterWithSetpoints.ipynb>`_.
+    `This notebook <../../examples/writing_drivers/A-ParameterWithSetpoints-Example-with-Dual-Setpoints.ipynb>`_.
```
Comment (astafan8) [inline qcodes/instrument/parameter.py]: shouldn't it be this https://github.com/QCoDeS/Qcodes/blob/master/docs/examples/Parameters/Simple-Example-of-ParameterWithSetpoints.ipynb ?

Comment (astafan8) [reply qcodes/instrument/parameter.py]: or just both :)

Comment (lakhotiaharshit) [reply qcodes/instrument/parameter.py]: Maybe both. I was also confused about what would be a better choice.

Comment (astafan8) [reply qcodes/instrument/parameter.py]: the SimpleExample is definitely the preferred choice. But let's have both.

**Comment by astafan8** [comment]
CI is complaining :)

**Pull request merged**